lib/lita/handlers/cmd.rb

```diff
@@ -77,13 +81,17 @@ def get_script_list(resp, config)
         bot = Lita::Robot.new
         auth = bot.auth
 
-        list = Dir.entries(config.scripts_dir).select { |f| File.file? "#{config.scripts_dir}/#{f}" }
+        list = Dir.entries(config.scripts_dir).select do |f|
+          (File.file? "#{config.scripts_dir}/#{f}") &&
+          (f.match(config.ignore_scripts))
```

Comment:
Shouldn't this be an `&& !(f.match...`? The way this is currently it should _only_ return files that contain `~` or `#`.